x-pack/plugins/translations/translations/ja-JP.json

```diff
@@ -31226,6 +31226,7 @@
     "unifiedSearch.filter.filterBar.pinFilterButtonLabel": "すべてのアプリにピン付け",
     "unifiedSearch.filter.filterBar.pinnedFilterPrefix": "ピン付け済み",
     "unifiedSearch.filter.filterBar.unpinFilterButtonLabel": "ピンを外す",
+    "unifiedSearch.filter.filterBar.invalidDateFormatProvidedErrorMessage": "無効な日付形式が提供されました",
```

Comment:
Why have you added these translations? You don't need to add anything here when you are adding a new translation property.

Reply:
It seemed to me a good idea to add translations into different languages. Okay, I'll delete them now.

Reply:
The translations team will handle this :)